docs/how-to/slice-a-package.md

```diff
@@ -0,0 +1,626 @@
+
+        <details>
+        <summary>Example: Spread test for OpenSSL</summary>
+
+        ```yaml
+        summary: Integration tests for openssl
+
+        execute: |
+          rootfs="$(install-slices openssl_bins)"
+ 
+          chroot "$rootfs" openssl --help
+ 
+          test_sha1="4e1243bd22c66e76c2ba9eddc1f91394e57f9f83"
+          chroot "$rootfs" openssl sha1 <<< "test" | grep $test_sha1
+        ```
+
+        </details>
+
+    2. [ ] **Run your Spread test**
+
+        Run the test with `spread`:
+
+        ```{terminal}
+        :input: spread lxd:tests/spread/integration/openssl
+
+        ...
+        2025-04-14 18:29:40 Preparing lxd:ubuntu-oracular:tests/spread/integration/openssl (lxd:ubuntu-oracular)...
+        2025-04-14 18:29:40 Executing lxd:ubuntu-oracular:tests/spread/integration/openssl (lxd:ubuntu-oracular) (1/1)...
+        2025-04-14 18:29:50 Discarding lxd:ubuntu-oracular...
+        2025-04-14 18:29:52 Successful tasks: 1
+        2025-04-14 18:29:52 Aborted tasks: 0
+        ```
+
+19. [ ] **Contribute!**
+
+    New slice definitions are welcome! Please contribute your new slices to the
+    {{chisel_releases_repo}}. See the [contributing guide](https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md).
+
+(create_sdf_examples)=
+
+## Example: good vs bad slice definitions
+
+`````{tab-set}
+````{tab-item} Good
+:class-label: sd-text-info
+
+```yaml
+package: libpython3.12-stdlib
+
+essential:
+  - libpython3.12-stdlib_copyright
+
+slices:
+  # File Formats
+  # https://docs.python.org/3.12/library/fileformats.html
+  file-formats:
+    essential:
+      - libpython3.12-stdlib_core
+      - libpython3.12-stdlib_frameworks
+      - libpython3.12-stdlib_markup-tools
+    contents:
+      /usr/lib/python3.12/netrc.py:
+      /usr/lib/python3.12/plistlib.py:
+      /usr/lib/python3.12/tomllib/**:
+      /usr/lib/python3.12/xdrlib.py:
+
+  ...
+```
+
+````
+
+````{tab-item} Bad
+:class-label: sd-text-danger
+
+```yaml
+package: libpython3.12-stdlib
+
+# Missing a global "essential" dependency on the "copyright" slice, thus
+# forcing it to be an "essential" in every slice below, making the file
+# less readable and more prone to missing that dependency.
+
+slices:
+  # Controversial grouping by type. Makes more sense to group Python by function.
+  modules:
+    # Possibly missing the dependencies on ".so" files and other contents.
+
+    # No sorting of contents
+    contents:
+      # No need to define this symlink. It is already defined in the deb
+      /usr/share/doc/libpython3.12-stdlib: {symlink: /usr/share/doc/libpython3.12-minimal}
+
+      # Chose to group by type, but still mixing types within the same slice
+      /usr/lib/python3.12/netrc.py:
+      /usr/lib/python3.12/pydoc_data/_pydoc.css
+
+      # Bad glob, possibly conflicting with other python packages
+      /usr/lib/python3.*/tomllib/**:
+
+  ...
+```
+```
```

Comment:
This probably leads to an empty code block at the end of the "Bad" syntax.

![image](https://github.com/user-attachments/assets/7a7da140-7bc8-41cd-b0eb-81effbfd4f2e)

Reply:
indeed. should be ok now